man/tpm2_policycphash.1.md

```diff
@@ -26,7 +26,7 @@ in a TPM command is checked against the one specified in the policy.
     The policy session file generated via the **-S** option to
     **tpm2_startauthsession**(1).
 
-  * **\--cphash**=_FILE_:
```

Comment:
keep the old one and say that its deprecated, like:

tpm2_makecredential.1.md:39:    **DEPRECATED**, use **-u** or **--public** instead.

Add the new option as well.

Reply:
fixed